app/Components/templates/VotingStateBox.latte

```diff
@@ -1,24 +1,44 @@
 {varType Model\Vote\VotingResult $votingResult}
 {varType Model\Vote\VotingTime $votingTime}
+{varType int $delegatesCount}
 {varType bool $showResult}
+{varType bool $canEdit}
 
 {define resultRows, $header, $candidates, $votersCount}
     <tr>
         <th>Pořadí</th>
         <th>{$header} (hlasovalo {$votersCount})</th>
         <th>Hlasů</th>
         <th>Výsledek</th>
+        <th n:if="$canEdit">Pořadí</th>
     </tr>
     <tr n:foreach="$candidates as $c">
-        <td n:inner-if="! $c->isRemovedFromOrder()">{$iterator->getCounter ()}</td>
+        {var $i = $iterator->getCounter() - 1}
```

Comment:
díky tomu pak bude $i ==0 a dole se pak vola $candidates[$i - 1], coz by znamenalo index -1

Reply:
uz vidim, ze tam mas to isFirst()